src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+// Colors
+$color-transition: color 300ms;
+$transform-transition: transform 300ms;
```

Comment:
I think you can replace those variables with just one:
`$transition-duration: 300ms;`

And use it like this:
`transition: color $transition-duration, transform $transition-duration;`

Reply:
agree